scheduler/src/cook/kubernetes/api.clj

```diff
@@ -92,13 +90,28 @@
                                               .getMetadata
                                               .getName))
                                         (.getItems current-pods))]
-    (log/info "Updating all-pods-atom with pods" (keys pod-name->pod))
-    (reset! all-pods-atom pod-name->pod)
-    (reset! cook-pods-atom
-            (->> pod-name->pod
-                 (filter (fn [[_ pod]] (is-cook-scheduler-pod pod)))
-                 (into {})))
-    (log/info "Updating cook-pods-atom with pods" (keys @cook-pods-atom))
+    [current-pods pod-name->pod]))
```

Comment:
What about just returning `pod-name->pod`? You can get `current-pods` by looking at the values of the map.

Reply:
current-pods is a V1PodList. We need that in order to get its resource version to setup the watch. Everything else just wants the map.

Reply:
👍